docs/mutation-data-transcript-annotation.md

```diff
@@ -0,0 +1,22 @@
+## Mutation Data Transcript Annotation
+This document describes how each mutation in cBioPortal gets annotated with a specific gene symbol + protein change.
+
+### Biological Background
+This section explains the concepts of protein isoforms and transcripts.
+
+#### What is an isoform?
+From a single gene (string of nucleotides) multiple protein sequences can be formed (string of amino acids). For example: parts of the gene that code for proteins (exons) can be included or excluded. Each of these different proteins is called an [isoform](https://en.wikipedia.org/wiki/Protein_isoform). A single mutation can change the amino acids in each of these these isoforms differently. E.g. in one isoform it might change a P to a T, but in the other isoform that particular exon does not get included and it is therefore not changing the amino acid sequence at all. In cBioPortal for convenience sake we assign a single gene symbol + protein change to each mutation. In several cases this works well because there is only one protein isoform relevant in a clinical setting. There are of course exceptions and we are therefore working on improving this representation. The term isoform generally refers to a protein, but oftentimes one wants to know changes to non-exonic parts of the gene as well. Although cBioPortal currently only displays exonic events aside from a few exceptions of known oncogenic events (e.g. TERT promoter), we do store the change to the transcript in the MAF file. This allows computational researchers to easily investigate these further outside of cBioPortal.
```

Comment:
> The term isoform generally refers to a protein, but oftentimes one wants to know changes to non-exonic parts of the gene as well. Although cBioPortal currently only displays exonic events aside from a few exceptions of known oncogenic events (e.g. TERT promoter), we do store the change to the transcript in the MAF file. This allows computational researchers to easily investigate these further outside of cBioPortal.

Do we need this information? It doesn't seem relevant to the following paragraphs. I was anticipating some sort of added complexity for non-exonic events, but you never mentioned it again.

Reply:
good point - i removed it